### ISSUE TITLE
not installing

### ISSUE THREAD

**Issue by 3Dprinted1** [JoeJoeTV/AstroTuxLauncher#2]
python3 AstroTuxLauncher.py install
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.25.8) or chardet (5.0.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Traceback (most recent call last):
  File "AstroTuxLauncher.py", line 25, in <module>
    from astro.dedicatedserver import AstroDedicatedServer, ServerStatus
  File "/home/ben/AstroTuxLauncher/astro/dedicatedserver.py", line 138, in <module>
    class DedicatedServerConfig:
  File "/home/ben/AstroTuxLauncher/astro/dedicatedserver.py", line 162, in DedicatedServerConfig
    PlayerProperties: list[PlayerPropertiesEntry] = field(default_factory=list, metadata=pp_list_field)
TypeError: 'type' object is not subscriptable

**Comment by JoeJoeTV** [comment]
Which python version are you using?

(You can get that by executing `python3 --version`)

**Comment by JoeJoeTV** [comment]
I have looked at this again with a python 3.8 installation and it seems like some features of some modules that I use were only added in Python 3.9, so this is the minimum version that should be used to run AstroTuxLauncher.

For the future, I have added an error message, if AstroTuxLauncher is run with a lower python version.

P.S. On Ubuntu 20.04 and derivatives, Python 3.9 can be installed along side Python 3.8 by simply installing the `python3.9` package
